ostree/ostree_dest.go

```diff
@@ -74,6 +158,9 @@ type ostreeImageDestination struct {
 	repo          *C.struct_OstreeRepo
 }
 
+// map to hold references to the required libselinux functions
+selinuxFunctions := make(map[string]unsafe.Pointer)
```

Comment:
(FWIW this could use an integer index (e.g. a typed enum) instead of strings, for better performance and to make sure no typos happen.)

(But still I’d strongly prefer a build tag.)